packages/lodestar/test/perf/chain/validation/attestation.test.ts

```diff
@@ -26,3 +27,46 @@ describe("validate gossip attestation", () => {
     });
   }
 });
+
+describe.only("test throw AttestationError vs object", () => {
+  itBench({
+    id: "AttestationError",
```

Comment:
The `id` of the benchmark must be globally unique in the entire Lodestar monorepo. This ID will be displayed in PRs to it needs more context. For example:
- `Reject AttestationError x1000`
- `Reject AttestationError x1`
- `Reject plain object x1000`
- `Reject plain object x1`